src/containers/RepositoriesDetailContainer/RepositoriesDetailDuck.js

```diff
@@ -0,0 +1,75 @@
+import ApiUtils from '../../utils/ApiUtils.js';
+
+// Actions
+const LOAD = 'react-redux-starter-app/repositoriesDetail/LOAD';
+const LOAD_SUCCESS = 'react-redux-starter-app/repositoriesDetail/LOAD_SUCCESS';
+const LOAD_ERROR = 'react-redux-starter-app/repositoriesDetail/LOAD_ERROR';
+
+const initialState = {
+  isLoading: false,
+  repos: undefined
+};
+
+// Reducers
+const REDUCERS = {
+  [LOAD]: (state, action) => ({
+    ...state,
+    isLoading: true
+  }),
+  [LOAD_SUCCESS]: (state, action) => ({
+    ...state,
+    isLoading: false,
+    repos: action.data.map(repo => ({
+      author: repo.owner.login,
+      avatar: repo.owner.avatar_url,
```

Comment:
how do we ensure that the data in repo that we receive is consistent?
e.g. are we 100% sure that `repo.owner` object is going to be there?